src/pages/Metrics/Metrics.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { PageHeaderWrapper } from '@ant-design/pro-layout';
+import { Empty, Button, Card } from 'antd';
+import { history } from 'umi';
+import { stringify } from 'qs';
+
+const Metrics: React.FC<{}> = () => {
```

Comment:
,

Reply:
empty typing should be remove

Reply:
should I remove `React.FC<{}>` in `pages/404.tsx` `pages/Setting/Setting.tsx` file?

Reply:
yep but not in this PR, those files were generated by Ant Design Pro.